protoc_plugin/CHANGELOG.md

```diff
@@ -27,7 +31,7 @@
 
   `_Foo` will get the name `Foo_` and `Foo_` will now end up being called `Foo__`.
 
-## 0.11.0
+## 11.0.0
 
```

Comment:
You could add a comment here explaining that the jump in version number is due to a reviewer oversight :)